src/System.Reflection.Metadata/specs/Ecma-335-Issues.md

```diff
@@ -0,0 +1,133 @@
+# Known Issues in ECMA-335 CLI Specification 
```

Comment:
It would be great to have a document where we could share issues more broadly - i.e. beyond those encountered in this context. Perhaps it belongs in the documentation directory of the coreclr repo (and yes, I realize that I'm probably the one that should create that ...)